uber/models.py

```diff
@@ -975,7 +975,7 @@ class Attendee(MagModel, TakesPaymentMixin):
     games  = relationship('Game', backref='attendee')
     shifts = relationship('Shift', backref='attendee')
     checkouts = relationship('Checkout', backref='attendee')
```

Comment:
I also fixed this in the other places in the same file where I deliberate set the value to ``all`` thinking that didn't include ``delete``.